test/fakeintake/README.md

```diff
@@ -4,52 +4,52 @@ Exposes a catch-all API for Datadog Agent POST requests.
 
 ## Requirements
 
-* [Golang 1.19](https://go.dev/dl/)
```

Comment:
Any reason to downgrade?

Reply:
It seems that `go.mod` was not updated but `datadog-agent` is now built with `1.19`